Sources/SwiftIndexStore/SwiftIndexStore.swift

```diff
@@ -123,6 +133,7 @@ public final class IndexStore {
                 throw error
             }
         }
+        lib.record_reader_dispose(reader)
```

Comment:
This line won't be reached when the above lines throw an error, so please use `defer`

Reply:
Thanks for pointing that out